src/DataStructures/Tensor/Tensor.hpp

```diff
@@ -437,6 +437,25 @@ class Tensor<X, Symm, IndexList<Indices...>> {
   std::pair<std::vector<std::string>, std::vector<X>> get_vector_of_data() const
       noexcept;
 
+  /*!
```

Comment:
How about a free function that takes a container and does the resize for each member? That seems more flexible. This should also have a test.

Reply:
Sounds good, I'll move it to utilities, then. Do you have any trouble with this implementation that assumes compatibility with range-based iteration?

Reply:
I personally don't, no. With the STL we wrap the iterator based algorithms because it's way more convenient to just be able to pass the container. However, if you think you'll need this capability in something like `std::unordered_map` you probably want an iterator-based implementation and a container wrapper. The reason being that range-based loops for maps would give a key-value pair (same with iterators, but there are transform iterators that could be used). So do what you will with that information, I'm happy either way :)